components/clarinet-deployments/src/lib.rs

```diff
@@ -296,12 +306,30 @@ fn handle_emulated_contract_call(
     result
 }
 
+// Main function that always includes boot contracts by default
 pub async fn generate_default_deployment(
     manifest: &ProjectManifest,
     network: &StacksNetwork,
     no_batch: bool,
     file_accessor: Option<&dyn FileAccessor>,
     forced_min_epoch: Option<StacksEpochId>,
+) -> Result<(DeploymentSpecification, DeploymentGenerationArtifacts), String> {
+    generate_default_deployment_with_boot_contracts(
+        manifest,
+        network,
+        no_batch,
+        file_accessor,
+        forced_min_epoch,
+    )
+    .await
+}
+
+pub async fn generate_default_deployment_with_boot_contracts(
```

Comment:
Do we need this extra function?
I think it can be removed and just keep `generate_default_deployment`